export const assertNever = (): never => {
  throw new Error('An error has occurred');
};
